### PR TITLE
Reword "Max level" to "Max severity level"

### LINKED ISSUE #421: Rename Max level selector
Rename to "Max severity level" for more clarity.

### PR DESCRIPTION
**Pull request by stveit**
Fixes #421

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ This file documents all changes to Argus-frontend. This file is primarily meant
 - Order of events and acknowledgements in feed in detailed incident view. Order is now oldest-first.
 - Made docker-compose build more flexible.
 
+- Change the label for the severity level selector from "Max level" to "Max severity level"
```
Comment (podliashanyk) [inline CHANGELOG.md:24]: This one can be repeated in NOTES.md as well

**Review by podliashanyk — CHANGES_REQUESTED** [review]
Short and sweet, just update NOTES.md as well 😌

**Review by hmpf — CHANGES_REQUESTED** [review]
Conflict in CHANGELOG, otherwise OK.

**Review by podliashanyk — APPROVED** [review]
👍🏻

**Pull request merged**